config/config.example.yml

```diff
@@ -1,11 +1,147 @@
+# Invidious configuration file
+---
+# Number of threads to use for crawling videos from channels
 channel_threads: 1
+# Number of threads to use for refreshing subscription feeds
 feed_threads: 1
+
+# Log file path or STDOUT
+output: STDOUT
+
+# Log verbosity
+# All   = 0, Trace = 1
+# Debug = 2, Info  = 3
+# Warn  = 4, Error = 5
+# Fatal = 6, Off   = 7
+log_level: 3
+
+# Database configuration
 db:
   user: kemal
   password: kemal
   host: localhost
   port: 5432
   dbname: invidious
+
+# Use polling to keep decryption function up to date
+decrypt_polling: true
+
+# Crawling channel videos, threads should refresh
+# all videos uploaded by a channel
 full_refresh: false
+
+# Define if accessed via HTTPS
 https_only: false
-domain:
+
+# Signing key for CSRF tokens
+# randomly generated by default
+#hmac_key: 
+
+# External domain
+domain: replace.me
```

Comment:
Should be empty for users running it locally.

Reply:
Might be good to point out that storyboards need this to be set and for local (non Docker) instances `domain: localhost` works.

Reply:
Then I guess it should be set to localhost.

Reply:
I don't think that works for Docker.